modules/ROOT/pages/_partials/commons/common-querybuilder.adoc

```diff
@@ -19,73 +19,56 @@
 
 // END::Local page attributes
 
-NOTE: The examples used in this topic are based on the _Travel Sample_ app and data introduced in the {couchbase-tutorials--mobile-workshop--xref} tutorial
+NOTE: The examples used here are based on the _Travel Sample_ app and data introduced in the {couchbase-tutorials--mobile-workshop--xref} tutorial
 
 == Introduction
 
-Couchbase Lite for {param-title} provides two ways to build and run database queries; the QueryBuilder API described in this topic and the {query-n1ql-mobile--xref}.
+Couchbase Lite for {param-title} provides two ways to build and run database queries; the QueryBuilder API described in this topic and {query-n1ql-mobile--xref}.
 
-Database queries defined with the QueryBuilder API use query statements of the form shown in <<ex-query-form>>.
-The structure and semantics of the query format are based on that of Couchbase's xref:server:learn:data/n1ql-versus-sql.adoc[{sqlpp_cbm} query language].
+Database queries defined with the QueryBuilder API use the query statement format shown in <<ex-query-form>>.
+The structure and semantics of the query format are based on Couchbase's xref:server:learn:data/n1ql-versus-sql.adoc[{sqlpp} query language].
 
 [#ex-query-form]
 .Query Format
 ====
 [source, SQL, subs="+attributes, +macros"]
 ----
-SELECT {blank-field} // <.>
-FROM 'database' // <.>
-WHERE {blank-field}, // <.>
-JOIN {blank-field} // <.>
-GROUP BY {blank-field} // <.>
-ORDER BY {blank-field} // <.>
+SELECT {blank-field} // 
```

Comment:
if you're getting rid of the `<.>` marker, get rid of the preceding comment marker too (e.g. the `//`) as I think that will be displayed otherwise, and isn't needed.